### DISCUSSION TITLE
post/new-backup-and-recover-solution/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#4] [Announcements]
# post/new-backup-and-recover-solution/

很久很久以前 Blog 上有一篇关于我如何备份文件的文章. 现在情况又发生了些变化, 经过不断的迭代, 我的备份方案有了不少的变化, 所以再来写一篇. 这次主要分享下我作为强迫症用户如何保持 Windows 系统”整洁”. Windows不知道大家会不会遇到不小心在电脑上装了流氓软件之后总觉得卸载不干净的情况. 之前还没这么强迫症, 但上了大学之后肯定是吃饱了撑的没事干就越来越强迫症了. 希望能保

https://blog.lyc8503.site/post/new-backup-and-recover-solution/

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。

**Comment by SuperKenVery** [comment]
妈的，感觉iOS就是阻止我学习这些的一大拦路虎啊啊啊啊啊
